src/validation.cpp

```diff
@@ -5023,12 +5024,21 @@ bool LoadMempool(CTxMemPool& pool)
         for (const auto& i : mapDeltas) {
             pool.PrioritiseTransaction(i.first, i.second);
         }
+
+        std::set<uint256> unbroadcast_txids;
```

Comment:
Perhaps this section for deserializing unbroadcast txids can be moved to a separate `try` ... `catch` block, so that the normal "Imported ..." message appears rather than a deserialization error?

Reply:
is this in regards to preventing one-time error when upgrading? or more general possible errors?

Reply:
Ah, it seems the return value of LoadMempool is actually ignored in case of failure, so it's indeed just a warning issue. It would seem cleaner that LoadMempool returns either returns true for successfully loaded (whether unbroadcast_txids was present or not), and false + empty mempool otherwise. That's just code esthetics though, no need to address it here.